README.md

```diff
@@ -31,14 +31,81 @@ using `make requirements` are kept up to date in latest `master` of those reposi
 
 If new dependencies were added in the `requirements.txt` of that
 repo that are not in the FPF PyPI mirror, then the maintainer needs
-to:
-
-1. Build those wheels using `make build-wheels`
-2. Push the tarball and wheel package of the new
-dependency to the FPF PyPI mirror using the steps described [here](https://github.com/freedomofpress/securedrop-debian-packaging-guide/issues/6).
-3. Make a PR updating the shasums and signature in this repository.
-4. Once this is done, `make requirements` can be used to update `build-requirements.txt`
-in the repository to be packaged.
+to do the following (we are taking `securedrop-client` project as example):
+
+### 1. Sync the wheels locally
+
+Sync all of the latest wheels `make syncwheels`
```

Comment:
there is no `make syncwheels` target in this repo. Do you mean `make fetch-wheels`, or am I missing something?